filecoin-proofs/tests/api.rs

```diff
@@ -614,75 +619,89 @@ fn seal_lifecycle_upgrade<Tree: 'static + MerkleTreeTrait<Hasher = TreeRHasher>>
 
 #[test]
 #[ignore]
-fn test_seal_proof_aggregation_1_2kib_porep_id_v1_1_base_8() -> Result<()> {
-    let proofs_to_aggregate = 1; // Requires auto-padding
-
-    let porep_id_v1_1: u64 = 5; // This is a RegisteredSealProof value
-
-    let mut porep_id = [0u8; 32];
-    porep_id[..8].copy_from_slice(&porep_id_v1_1.to_le_bytes());
-    assert!(!is_legacy_porep_id(porep_id));
-    aggregate_seal_proofs::<SectorShape2KiB>(SECTOR_SIZE_2_KIB, &porep_id, proofs_to_aggregate)
-}
-
-#[test]
-#[ignore]
-fn test_seal_proof_aggregation_3_2kib_porep_id_v1_1_base_8() -> Result<()> {
-    let proofs_to_aggregate = 3; // Requires auto-padding
+fn test_seal_proof_aggregation_2kib() -> Result<()> {
+    let test_inputs = vec![
+        (1, 5, ApiVersion::V1_1_0, vec![]),
+        (5, 5, ApiVersion::V1_2_0, vec![ApiFeature::SyntheticPoRep]),
+        (
+            257,
```

Comment:
Just out of curiosity, was aggregation of 257 test-sector PoRep Groth16 proofs chosen for a specific reason?

Reply:
Any of the non-pow2 sizes are testing padding to pow2.  We had an existing test that used that value, so it was replaced with an equivalent.

Reply:
Ok, sounds good. I was curious because (I believe that) this is saying to aggregate 257 NI-PoRep proofs, so 3341 Groth16 proofs (each NI-PoRep proof containing 13 Groth16/partition proofs), and if 257 was chosen arbitrarily when maybe a number like 160 would result in the same SnarkPack padding (i.e. `ceil(log2(257 * 13)) == ceil(log2(160 * 13)) == 12`).

Reply:
Good call on the count -- the FIP actually specifies bounds that I'll add